oteljava/all/src/test/scala/org/typelevel/otel4s/oteljava/metrics/ObservableSuite.scala

```diff
@@ -144,7 +144,7 @@ class ObservableSuite extends CatsEffectSuite {
           .withUnit("unit")
           .withDescription("description")
           .createWithCallback(
-            _.record(1234, Attribute[Boolean]("is_false", true))
+            _.record(1234, Attribute("is_false", true))
```

Comment:
wait, does this additional complexity somehow make the user experience *nicer*? (still pondering these changes)

Reply:
Fair. It brings additional complexity for no reason.